target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -1,4 +1,11 @@
+<<<<<<< HEAD
```

Comment:
Looks like there was a merge conflict here. I'm not too familiar with how the maven compiler works but you might want to take a look and see if there's a way to resolve this or what it was previously. 

Looks like when it compiles as yours all the META-INF files get deleted, which should be okay. Since Marifer's is the version with the META-INF files, maybe try keeping just yours? I would be careful that you're not accidentally reverting/deleting code though.